docs/source/concepts/provenance.rst

```diff
@@ -320,66 +320,72 @@ Let us first focus on the data provenance only (i.e., only ``input_calc`` and ``
+   :scale: 80%
 
-But what if there are more calculation called by |W_1|?
-These won't be deleted by the default behavior, because ``call_calc_forward=False``.
-This can be illustrated by considering what would happen if we targeted the workflow node |W_1| instead (which might be the most natural thing to do for what we intend to achieve):
+In this case, the node |C_1| would first be included because the data node |D_3| is being targeted (``create_reverse=True``), and this in turn would include the node |W_1| (``call_calc_reverse=True``) and then its parent workflow |W_0| (``call_work_reverse=True``).
+Then nodes |W_2|, |C_2| and |D_4| will be included because |W_0| was included, for the same reasons that were explained in the paragraphs above.
 
-.. _delexp_example02b:
-.. image:: include/images/delexp_example02-b00.png
-   :scale: 80%
 
-As you see, only |W_0| and |W_1| have been deleted, but |C_1| is still in the database.
-In this case in particular, to achieve the same result as above, it would suffice to enable ``call_calc_forward=True`` to traverse the ``call_calc`` link from |W_1| to |C_1| and then recover our desired result, starting from a different target (|W_1| here, instead of |C_1| above).
-The second workflow |W_2| would still be unaffected because there was no need to forward traverse any ``call_work`` link so far.
+Customizing the section to delete (or export)
+.............................................
 
-.. _delexp_example02c:
-.. image:: include/images/delexp_example02-c00.png
+This dependency between nodes becomes particularly relevant when, for example, a user with more knowledge of the internal procedures of the parent workflow |W_0| wants to only delete the calculations and results associated to workflow |W_1|.
+The intuitive action of targeting |W_1| does not produce the desired outcome:
+
+.. _delexp_example02-b01:
+.. image:: include/images/delexp_example02-b01.png
    :scale: 80%
 
-But what if some of the child processes of |W_1| are workflows instead of calculations?
-The naive answer would be to enable ``call_work_forward=True`` as well. However, this will delete much more that you might want! In fact, since we are also deleting |W_0|, this last rule would also imply going through the ``call_work`` link between |W_0| and |W_2|, thus producing the following final undesired result, where most nodes have been deleted:
+Indeed |C_1| and |D_4| will be deleted (through ``call_calc_forward`` from |W_1| to |C_1| and ``create_forward`` from |C_1| to |D_3|), but so will |W_0| (through ``call_work_reverse`` from |W_1|), |W_2| (``call_work_forward`` from |W_0|), |C_2| (``call_calc_forward`` from |W_2|) and |D_4| (``create_forward`` from |C_2|).
+The way to achieve the desired outcome is not at all obvious, although in this particular case one could propose some simple 'hacks', like targeting |W_1| setting the switchable flag ``call_work_forward=False`` (preventing the traversal from |W_0| to |W_2|):
```

Comment:
I did have a suggestion for this sentence as well that removed the "not at all obvious" and "hacks", which I would probably still prefer...

Reply:
I have now changed the "hack" and "not obvious" phrasings. I used a different version of what you proposed, not out of stuborness, but because I keep missing comments/corrections when github hides/colapses them (only now I have noticed it; this is also the reason why I didn't adress it in the previous commit). Luckily I think in this case I still adressed what you wanted change, correct? I will try to pay more attention to that in the future, apologies.

Reply:
thanks & no worries, happens to me as well!